contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3174,3 +3183,7 @@ END;
 $BODY$
 LANGUAGE plpgsql;
 GRANT EXECUTE ON FUNCTION sys.INDEXPROPERTY(IN object_id INT, IN index_or_statistics_name sys.nvarchar(128),  IN property sys.varchar(128)) TO PUBLIC;
+
+GRANT EXECUTE ON FUNCTION sys.power(int,int) TO PUBLIC;
+GRANT EXECUTE ON FUNCTION sys.radians(int) TO PUBLIC;
+GRANT EXECUTE ON FUNCTION sys.degrees(int) TO PUBLIC;
```

Comment:
Please put these GRANT statements right after each CREATE.

Reply:
Did the modifications